onnxruntime/core/providers/webnn/builders/helper.cc

```diff
@@ -69,26 +69,24 @@ bool IsNodeSupported(const Node& node, const GraphViewer& graph_viewer, const We
   }
 }
 
-bool IsInputSupported(const NodeArg& input, const std::string& parent_name, const logging::Logger& logger) {
-  const auto& input_name = input.Name();
-  const auto* shape_proto = input.Shape();
+bool IsNodeArgSupported(const NodeArg& node_arg, const std::string& parent_name, const logging::Logger& logger) {
```

Comment:
How about `IsNodeShapeSupported`?

Reply:
NodeShape sounds like the shape of a node, but it's actually the shape of the operands of a node...

Reply:
How about `IsInputOutputSupported`?

Reply:
Discussed off line, renamed it to `IsTensorShapeSupported`.